src/query/service/src/interpreters/interpreter_copy.rs

```diff
@@ -366,21 +398,78 @@ impl CopyInterpreter {
     #[async_backtrace::framed]
     async fn build_cluster_copy_into_table_pipeline(
         &self,
-        distributed_plan: &DistributedCopyIntoTable,
+        distributed_plan: &CopyPlanType,
     ) -> Result<PipelineBuildResult> {
-        // add exchange plan node to enable distributed
-        // TODO(leiysky): we reuse the id of exchange here,
-        // which is not correct. We should generate a new id for insert.
-        let exchange_plan = PhysicalPlan::Exchange(Exchange {
-            plan_id: 0,
-            input: Box::new(PhysicalPlan::DistributedCopyIntoTable(Box::new(
-                distributed_plan.clone(),
-            ))),
-            kind: FragmentKind::Merge,
-            keys: Vec::new(),
-        });
-        let build_res = build_distributed_pipeline(&self.ctx, &exchange_plan, false).await?;
+        let catalog_name;
+        let database_name;
+        let table_name;
+        let stage_info;
+        let files;
+        let force;
+        let purge;
+        let is_overwrite;
+        let mut build_res = match distributed_plan {
```

Comment:
I think you can merge these `let`s into one `let` to make codes prettier.

Reply:
thanks